config/src/network_id.rs

```diff
@@ -146,17 +155,12 @@ impl fmt::Display for NetworkId {
 const VFN_NETWORK: &str = "vfn";
 
 impl NetworkId {
-    /// Convenience function to specify the VFN network
-    pub fn vfn_network() -> NetworkId {
-        NetworkId::Private(VFN_NETWORK.to_string())
-    }
-
     pub fn is_vfn_network(&self) -> bool {
-        matches!(self, NetworkId::Private(network) if network == VFN_NETWORK)
+        self == &NetworkId::Vfn
     }
 
     pub fn is_validator_network(&self) -> bool {
-        matches!(self, NetworkId::Validator)
+        self == &NetworkId::Validator
```

Comment:
small nit (probably not worth fixing) Using a match statement or `matches`! macro may be a bit more idiomatic